README.md

```diff
@@ -1 +1,13 @@
-Elrond SDK for Dart and Flutter
\ No newline at end of file
+Elrond SDK for Dart and Flutter
+
+## What it is 
+
+[Elrond](https://elrond.com/) is a highly scalable, fast and secure blockchain platform for distributed apps, enterprise use cases and the new internet economy.
+
+This SDK is a [Dart](https://dart.dev/) library for building decentralized applications on [Elrond](https://elrond.com/) blockchain.
+
+It allows interaction with Elrond nodes to send transactions, interact with smart contracts and much more!
+
+## License
+
+[MIT](https://github.com/moneytrackio/tezart/blob/main/LICENSE)
```

Comment:
we should redirect to the license inside the repository instead

Reply:
Sorry for my copy paste